app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask, request, jsonify
+app = Flask(__name__)
+@app.route("/drone-service", methods = ["GET"])
```

Comment:
also leave a line between the app instantiation and the endpoints.. this will make the code look more organized